app.js

```diff
@@ -1,9 +1,29 @@
 'use strict';
 
 const herokuCluster = require('@quarterto/heroku-cluster');
+const https = require('https');
+const denodeify = require('denodeify');
+const path = require('path');
+const fs = require('fs');
 const app = require('./server');
 
-herokuCluster({
-	defaultPort: 5000,
-	app,
-});
+const isDevelopment = app.get('env') === 'development';
+
+if(isDevelopment) {
+	const readFile = denodeify(fs.readFile);
+	const keyFile = readFile(path.resolve(__dirname, 'key.pem'));
+	const certFile = readFile(path.resolve(__dirname, 'cert.pem'));
+	Promise.all([keyFile, certFile])
+		.then(([key, cert]) => {
+			https.createServer({ key, cert }, app)
+				.listen(5050, () => console.log('App listening on port 5050 (https) in development'));
```

Comment:
ft-app uses `httpolyglot` to run http and https on the same port, which works really well http://git.svc.ft.com/projects/FTA/repos/ft-app/browse/server/index.js

Reply:
Ooh, this is much neater!